### DISCUSSION TITLE
Connecting to LYWSD03MMC: device.watchAdvertisements is undefined

### DISCUSSION THREAD

**Discussion by dcartman** [pvvx/ATC_MiThermometer#523] [Q&A] [answered]
I recently purchased 4 LYWSD03MMC devices, when attempting to connect to them I am receiving an error stating that device.watchAdvertisements is undefined.

New to this firmware and connection process, any ideas where I'm going wrong?

**Comment by pvvx** [comment, marked as answer]
![image](https://github.com/pvvx/ATC_MiThermometer/assets/12629515/ea203d45-0f00-4a66-a0e5-1fc7c5ee33a5)

and https://www.google.com/search?q=device.watchAdvertisements -> 
https://stackoverflow.com/questions/64654250/device-watchadvertisements-is-not-a-function-web-bluetoothchrome
